packages/gatsby-plugin-ts/readme.md

```diff
@@ -61,6 +61,16 @@ In order for this plugin to work right, you'd need to set your compile options l
 |options.fileName| `graphql-type.ts` | path to the generated file. By default, it's placed at the project root directory & it should not be placed into `src`, since this will create an infinite loop|
 |options.codegenDelay| `200` | amount of delay from file change to codegen|
 |options.pluckConfig| <pre>{ globalGqlIdentifierName: "graphql", modules: [ { name: 'gatsby', identifier: 'graphql' } ] }</pre> | options passed to [graphql-tag-pluck](https://github.com/ardatan/graphql-toolkit/tree/master/packages/graphql-tag-pluck) when extracting queries and fragments from documents |
+|options.additionalSchemas| <pre>[]</pre> | array of additional schemas (other that the schema used by gatsby queries) for which types should be generated for. This is useful e.g. when you use client-side queries (e.g. with apollo-client) where you are querying another schema/endpoint |
```

Comment:
I think this would be easier to maintain as just a link to `packages/gatsby-plugin-graphql-codegen/readme.md`. We can deep link with anchors (e.g. `thing/#anchor-id) for a better docs experience.